crystalplan.py

```diff
@@ -1,4 +1,4 @@
-#/usr/bin/env/python
+#!/usr/bin/env/ python
```

Comment:
This works, but it should be
```suggestion
#!/usr/bin/env python
```